chain/actors/actor_miner.go

```diff
@@ -419,36 +412,28 @@ func (sma StorageMinerActor) SubmitPoSt(act *types.Actor, vmctx types.VMContext,
 		return nil, aerrors.New(1, "not authorized to submit post for miner")
 	}
 
-	currentProvingPeriodEnd, _ := ProvingPeriodEnd(self.ProvingPeriodEnd, vmctx.BlockHeight())
-
-	feesRequired := types.NewInt(0)
-
-	if currentProvingPeriodEnd > self.ProvingPeriodEnd {
-		//TODO late fee calc
-		feesRequired = types.BigAdd(feesRequired, types.NewInt(1000))
-	}
-
-	//TODO temporary sector failure fees
-
-	msgVal := vmctx.Message().Value
-	if msgVal.LessThan(feesRequired) {
-		return nil, aerrors.New(2, "not enough funds to pay post submission fees")
-	}
+	/*
+		// TODO: handle fees
+		msgVal := vmctx.Message().Value
```

Comment:
should we just not worry about fees for now? It was kindof a placeholder anyways, and the spec isnt easy to understand here.

Reply:
Let's not